project-koku/masu
# Changing kafka msg validation from hash to request_id

Changing message validation body to `request_id`

*Testing*
1. Upload a payload to upload service with the masu worker listening for messages.
Masu Worker:
```
masu_worker    | [2019-05-21 20:54:28,201: INFO/MainProcess] Validating message: b'{"request_id": "52df9f748eabcfea", "validation": "success"}'

```
2. Verify that the upload service sends the payload to the `available` topic; confirming that it correctly received our validation message
Upload Service:
```
upload-service_1  | MainThread INFO tornado.access - 202 POST /api/ingress/v1/upload (172.27.0.1) 3.96ms
upload-service_1  | MainThread INFO upload-service - request_id [52df9f748eabcfea] uploaded! elapsed [0.001750sec] url [/tmp/uploads/insights-perm/52df9f748eabcfea]
upload-service_1  | MainThread INFO upload-service - Data for request_id [52df9f748eabcfea] to topic [platform.upload.hccm] put on produce queue (qsize now: 1)
upload-service_1  | MainThread INFO upload-service - Popped data from produce queue (qsize now: 0) for topic [platform.upload.hccm], request_id [52df9f748eabcfea]
upload-service_1  | MainThread INFO upload-service - send data for topic [platform.upload.hccm] with request_id [52df9f748eabcfea] succeeded
upload-service_1  | MainThread INFO upload-service - Processing 1 messages from topic [platform.upload.validation]
upload-service_1  | MainThread INFO upload-service - processing message: payload [52df9f748eabcfea] - success
upload-service_1  | MainThread INFO upload-service - data for topic [platform.upload.available], request_id [52df9f748eabcfea], inv_id [None] put on produce queue (qsize now: 1)
upload-service_1  | MainThread INFO upload-service - Popped data from produce queue (qsize now: 0) for topic [platform.upload.available], request_id [52df9f748eabcfea]
upload-service_1  | MainThread INFO upload-service - send data for topic [platform.upload.available] with request_id [52df9f748eabcfea] succeeded
```

Message we are looking for:
`upload-service_1  | MainThread INFO upload-service - send data for topic [platform.upload.available] with request_id [52df9f748eabcfea] succeeded`
